ext/node/polyfills/internal/fs/handle.ts

```diff
@@ -26,11 +26,15 @@ interface ReadResult {
   buffer: Buffer;
 }
 
+type Path =string | Buffer | URL;
```

Comment:
Nit: formatting is off. Did you run the formatter as outlined in the contribution guidelines https://docs.deno.com/runtime/contributing/#submitting-a-pr-to-deno ?